Patch the build system to support par2-turbo downloaded into the source
directory.
--- a/cmake/par2-turbo.cmake
+++ b/cmake/par2-turbo.cmake
@@ -27,11 +27,7 @@ endif()
 ExternalProject_add(
 	par2-turbo
 	PREFIX			par2-turbo
-	GIT_REPOSITORY	https://github.com/nzbgetcom/par2cmdline-turbo.git
-	GIT_TAG			v1.1.1-nzbget-20241128
-	TLS_VERIFY		TRUE
-	GIT_SHALLOW		TRUE
-	GIT_PROGRESS	TRUE
+	URL par2-turbo
 	DOWNLOAD_EXTRACT_TIMESTAMP	TRUE
 	CMAKE_ARGS		${CMAKE_ARGS}
 	INSTALL_COMMAND	""
